### PR TITLE
added more content to the readme

### PR DESCRIPTION
**Pull request by bergos**
Adds an overview of all object types and supported constructor arguments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,106 @@
...
+There are two subtypes of `Source`.
+A plain `Source` can't be used in a `View`.
+One of the subtypes must be used.
+
+Supported constructor arguments:
+
+- `endpointUrl`: SPARQL endpoint URL as string, RDF/JS `NamedNode` or Clownface object.
+- `user`: User for the SPARQL endpoint as string.
+- `password`: Password for the SPARQL endpoint as string.
+- `sourceGraph`: Named Graph as string, RDF/JS `NamedNode` or Clownface object.
```
Comment (l00mi) [inline README.md:26]: Is it possible to also query the union graph? Is this argument optional?

Comment (l00mi) [reply README.md:26]: It says so here https://github.com/zazuko/rdf-cube-view-query/pull/21/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15 i see. Lets repeat it hear for clarity?

**Review by l00mi — APPROVED** [review]
That is a good base. I think next step is to use a simple example to illustrate the use of the different Objects.

Do we want to provide a sample cube on an endpoint that the example can also be run?

File: README.md (hunk 1 above)
Comment (ktk) [reply README.md:26]: Also could multiple graphs be provided? I don't have a use-case now but could become a thing when we have versions.

Comment (bergos) [reply README.md:26]: I didn't add any optional info for the arguments at all. Will do it in a separate PR more focused on code. `undefined` or `null` means union graph, like in any other RDF/JS library. Multiple graphs are not supported. The graph is directly translated to a SPARQL query. Having multiple graphs would make it a little bit complicated.

Comment (l00mi) [reply README.md:26]: please merge

**Comment by bergos** [comment]
Some examples in the `examples` folder are using already the example cubes from the cube schema. Based on that example data I will add more code related stuff to the docs in a separate PR.

**Pull request merged**